lib/src/tests/Encoder.spec.ts

```diff
@@ -82,6 +82,12 @@ describe('Encoder', () => {
                 EncoderTestUtils.expectToBeBijective(encoder, disjunctedValue);
             }
         });
+
+        it('should fail if no matcher matches', () => {
+            const value: number | boolean = 4;
+            const wrongEncoder : Encoder<number | boolean> = Encoder.disjunction([is2, is2], [encoder1, encoder2]);
```

Comment:
rename "is2" et "envocder1" et "encoder2" then perhaps explain this to me like I'm five.
Or like I'm new to the code :D

Reply:
changé et commenté pour être plus clair, vois si tu comprends maintenant !